Doc/Tutorial/chapter_introduction.tex

```diff
@@ -312,6 +312,24 @@ \section{Frequently Asked Questions (FAQ)}
   \item \emph{Why doesn't} \verb|Bio.Fasta| \emph{work?} \\
   We deprecated the \verb|Bio.Fasta| module in Biopython 1.51 (August 2009) and removed it in Biopython 1.55 (August 2010). There is a brief example showing how to convert old code to use \verb|Bio.SeqIO| instead in the \href{https://github.com/biopython/biopython/blob/master/DEPRECATED.rst}{DEPRECATED.rst} file.
 
+  \item \emph{Why does Biopython start so many threads?} \\
+  Some parts of Biopython make use of \href{http://www.numpy.org}{NumPy}, which in turn makes use of a system-dependent BLAS (Basic Linear Algebra Subprograms) library. Some BLAS libraries automatically spawn one thread for each available CPU core. On systems where this is the case, simply importing NumPy (or any Biopython module relying on NumPy) will spawn several threads. The number of threads can be changed by setting an environment variable before NumPy is loaded: if your version of NumPy is linked to the Intel MKL, set \verb|MKL_NUM_THREADS| to the desired number; for OpenBLAS, set \verb|OMP_NUM_THREADS|; otherwise, consult the documentation for your BLAS library.
```

Comment:
Should we say *"... which in turn may make use of ..."* here (ie *may*)? After all, NumPy might be built with ATLAS instead of BLAS?